pdc/apps/release/forms.py

```diff
@@ -43,9 +43,9 @@ def get_query(self, request):
         query = Q()
 
         if search:
-            query |= Q(short__icontains=search)
-            query |= Q(version__icontains=search)
             query |= Q(name__icontains=search)
+            query |= Q(base_product_id__icontains=search)
+            query |= Q(release_type__short__icontains=search)
 
         # TODO: disabled
```

Comment:
Oops, some leftover comment. :)  (Blame is of course the "initial import" commit.)

Reply:
This was probably meant to allow searching by whether it's active. That is possible by filtering the table, so I'll add another commit to remove these comments.